packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -311,9 +311,10 @@ class DateTimePicker extends React.Component {
         isValid = true;
       }
 
+      const inputDateTime = `${iSOString.substr(0, 10)} ${this.timeValue}`;
       const options = {
         iSO: iSOString,
-        inputValue: value,
+        inputValue: inputDateTime,
```

Comment:
This used to be what ever value the user entered. This would break consumers since it will now be in the iSO format.

Reply:
This and the above are temporary. I'm currently working around the formatting issues, as well as composing the total input value for use in these functions (as it stands, it really only recognizes the year input).

Reply:
Changed back! [452f10b](https://github.com/cerner/terra-framework/pull/1004/commits/452f10bd2fac12ea3c50aff9d7b90d0cff1d3b01)